network/links.md

```diff
@@ -77,28 +85,36 @@ default the `-p` flag binds the specified port to all interfaces on
 the host machine. But you can also specify a binding to a specific
 interface, for example only to the `localhost`.
 
+```bash
     $ docker run -d -p 127.0.0.1:80:5000 training/webapp python app.py
+```
 
 This would bind port 5000 inside the container to port 80 on the
 `localhost` or `127.0.0.1` interface on the host machine.
 
 Or, to bind port 5000 of the container to a dynamic port but only on the
 `localhost`, you could use:
 
+```bash
     $ docker run -d -p 127.0.0.1::5000 training/webapp python app.py
+```
```

Comment:
Same here ("console" and "un-indent")

```suggestion
```console
$ docker run -d -p 127.0.0.1::5000 training/webapp python app.py
```
```